### ISSUE TITLE
CMI + MMOLib for MMOItems

### ISSUE THREAD

**Issue by minfiserver** [Zrips/CMI#8625]
### Description of Issue

I use need use CMI with MMoItems+MMoLib on my server. But this creates lags. After start my server lose TPS. When i disable damage indicators on MMOLib all works perfectly, but i need use MMOLib damage indicators. I try to disable damage indicators on CMI configuration, but it isn’t helpful 

### Version Information

```shell
Версия плагина смІ: 9.7.0.4 MySQL
[17:39:26 INFO]: CMILib: 1.4.6.1
[17:39:26 INFO]: Сервер: Purpur (2091) 1.20.2-RO. 1-SNAPSHOT-
[17:39:26 INFO]: СМІ Economic: Enabled Vault: 1.7.3-СМІ СмІ Chat: Enabled
[17:39:26 INFO]: Modules -> 51 enabled and
× 8 disabled:
× signEdit, schedule, tablist, ranks, votifier,
```

### Agreements

- [X] My server is supported by CMI.
- [X] My version of CMI at the time of this report is up to date.
- [X] I have searched the github and asked around before making this report.

**Comment by minfiserver** [comment]
https://spark.lucko.me/Xo42ExHE8G
My spark timings.

**Comment by minfiserver** [comment]
@Zrips will it be possible to fix it?

**Comment by minfiserver** [comment]
up! @Zrips

**Comment by minfiserver** [comment]
Fix with CMI-9.7.3.2
